src/pages/UnauthenticatedHome.tsx

```diff
@@ -54,6 +54,15 @@ export const UnauthenticatedHome = () => {
           view source code
         </a>{' '}
         or <a href="https://opencollective.com/ohn">support us financially</a>.
+        <br />
+        Follow us on{' '}
+        <a rel="me" href="https://floss.social/@sleepybike">
+          mastodon
```

Comment:
Maybe upper case M in Mastodon? I'm not sure.

Reply:
How about _Follow us in the Fediverse or on Facebook._ ?

Reply:
I just made the change, to make fewer back-and-forth-s ;). If everything looks good for you, just [squash] merge it after giving it a look - which is a role of a _Maintainer_ according to [C4 point 2.1.6](https://rfc.zeromq.org/spec/42/) (unless we want to change it).

---
For some better context, I'm quoting [ZeroMQ - The Guide](https://zguide.zeromq.org/docs/chapter6/#Preliminaries) (I like most the last two paragraphs)

> > The project SHOULD have clearly documented guidelines for code style.
>
> This is a protocol plug-in: insert code style guidelines here. If you don’t document the code style you use, you have no basis except prejudice to reject patches.
>
> > A “Contributor” is a person who wishes to provide a patch, being a set of commits that solve some clearly identified problem. A “Maintainer” is a person who merge patches to the project. Maintainers are not developers; their job is to enforce process.
>
> Now we move on to definitions of the parties, and the splitting of roles that saved us from the sin of structural dependency on rare individuals. This worked well in libzmq, but as you will see it depends on the rest of the process. C4 isn’t a buffet; you will need the whole process (or something very like it), or it won’t hold together.
>
> > Contributors SHALL NOT have commit access to the repository unless they are also Maintainers. Maintainers SHALL have commit access to the repository.
>
> What we wanted to avoid was people pushing their changes directly to master. This was the biggest source of trouble in libzmq historically: large masses of raw code that took months or years to fully stabilize. We eventually followed other ZeroMQ projects like PyZMQ in using pull requests. We went further, and stipulated that all changes had to follow the same path. No exceptions for “special people”.
> 
> > Everyone, without distinction or discrimination, SHALL have an equal right to become a Contributor under the terms of this contract.
> 
> We had to state this explicitly. It used to be that the libzmq maintainers would reject patches simply because they didn’t like them. Now, that may sound reasonable to the author of a library (though libzmq was not written by any one person), but let’s remember our goal of creating a work that is owned by as many people as possible. Saying “I don’t like your patch so I’m going to reject it” is equivalent to saying, “I claim to own this and I think I’m better than you, and I don’t trust you”. Those are toxic messages to give to others who are thinking of becoming your co-investors.
> 
> I think this fight between individual expertise and collective intelligence plays out in other areas. It defined Wikipedia, and still does, a decade after that work surpassed anything built by small groups of experts. For me, we make software by slowly synthesizing the most accurate knowledge, much as we make Wikipedia articles.

Having that said, I'm happy to get any code style and design comments, as I also hope to learn some TS from you. (to be made in a follow-up patches).